pipelines/rh-advisories/rh-advisories.yaml

```diff
@@ -427,7 +432,8 @@ spec:
         - name: data
           workspace: release-workspace
       runAfter:
-        - push-rpm-data-to-pyxis
+        - collect-pyxis-params
```

Comment:
Doesn't this depend on create-pyxis-image so the pyxis repository you are publishing exists? Also it is okay to publish a pyxis repository before the sources are even in quay?

Reply:
> Doesn't this depend on create-pyxis-image so the pyxis repository you are publishing exists? 

No, create-pyxis-image creates ContainerImage in Pyxis. The Repository object in Pyxis needs to exist before the pipeline is even started. It's even one of the tests in the konflux-release-data repo - you can't add a RPA unless the Repository object already exists.

>  Also it is okay to publish a pyxis repository before the sources are even in quay?

In most cases it will already be published, so in that case it makes no difference. In case it was not published yet, there are two scenarios:

1. Either there is already some previous image or images that are marked as published (but the repo was hidden) - so then you will suddenly publish the older image(s) without showing the new one yet. But I don't think it's likely - your intention would need to be to release a new image and at the same time publish some older ones that you hid until now. I really don't see why anybody would do that. And even if they did, the issue would only be that potentially if the release failed before creating the new pyxis image, you would only see the older images in the catalog.
2. Or there were no older published images. In that case I'm not sure what the effect of having a published repository with no published image is - most likely the catalog will not show anything until there is at least one image - otherwise, what would it show? So most likely no effect.

Reply:
Okay.. 2 is what I was concerned about, but if you think it is fine, okay with me

Reply:
I asked in [slack](https://redhat-internal.slack.com/archives/C04HWBWT4QP/p1729250161040319) just to be sure.

Reply:
They confirmed - nothing will get displayed if the repo is published, but there is no published image yet.

Reply:
Great, thanks. That could be commented, but probably no one would notice it anyways or think to look there, so hopefully we just remember that if it ever comes up

Reply:
I added a note to the task README (but not in the yaml - it seems unnecessary there).